neat/controller_neat.py

```diff
@@ -2,7 +2,7 @@
 
 
 # implements controller structure for player
-class player_controller(Controller):
+class PlayerController(Controller):
```

Comment:
How about something like "NeatController"?

Reply:
in task 2 we will have in the same file the PlayerController and the EnemyController. As they did in the file demo_controller. so I think we can keep the name PlayerController

Reply:
Current name is a bit confusing, it doesn't tell us anything about the implementation or what it does. Imagine you see it imported in some other part of the code - you don't know what it does just by the name alone. It should include some more information.

Reply:
we can call it PlayerControllerNeat and then in the future we will call the enemy controller "EnemyControllerNeat". 
So we can do the same for Deap, by calling it "PlayerControllerDeap"
What do you think about that?

Reply:
Yeah, that would work. I feel like "NNControllerNeat" and "NNControllerDeap" give us the most information

Reply:
Yes but if you look at the demo_controller that they gave us they have 2 one for the player and one for the enemy.
I changed it to PlayerControllerNeat. if it is fine you can merge

Reply:
Alright, that's fine. Change the name for neat controller and we're good